pootle/static/js/editor/components/RawFontTextarea.js

```diff
@@ -61,15 +61,21 @@ const RawFontTextarea = React.createClass({
     this.mousetrap.bind(REDO_SHORTCUT, this.handleRedo);
 
     const { isRawMode } = this.props;
-    this.rawFont = new RawFontAware(this._textareaNode, { isRawMode });
+    const isRtlMode = (
+      this.context.currentLocaleDir === 'rtl' && !this.props.isRawMode
+    );
+    this.rawFont = new RawFontAware(this._textareaNode, { isRawMode, isRtlMode });
     this.previousSnapshot = this.rawFont.setSnapshot({
       value: this.props.initialValue,
     });
   },
 
   componentWillReceiveProps(nextProps) {
     if (this.props.isRawMode !== nextProps.isRawMode) {
-      this.rawFont.setMode({ isRawMode: nextProps.isRawMode });
+      const isRtlMode = (
+        this.context.currentLocaleDir === 'rtl' && !nextProps.isRawMode
+      );
+      this.rawFont.setMode({ isRtlMode, isRawMode: nextProps.isRawMode });
```

Comment:
Just raising it because in https://github.com/translate/pootle/pull/5320/files#diff-851c26485ad1b4f8747a8134b4d30b87R142 the order is the reverse one.